pep-0001.txt

```diff
@@ -488,6 +490,7 @@ optional and are described below.  All other headers are required. ::
     PEP: <pep number>
     Title: <pep title>
     Author: <list of authors' real names and optionally, email addrs>
+  * Sponsor: <real name of core developer sponsoring>
```

Comment:
Beware that this requires a change to the pep2html.py script.

Reply:
Yep, that's the next step before merging. 😄 I just wanted to make sure the idea had traction and support before I did the coding work.

Reply:
https://github.com/python/peps/pull/911